app/assets/javascripts/sufia/user_search.js

```diff
@@ -0,0 +1,34 @@
+(function( $ ){
+
+  $.fn.userSearch = function( options ) {
+    // Create some defaults, extending them with any options that were provided
+    var settings = $.extend( { }, options);
+
+    var $container = this;
+
+    return this.each(function() {
+      $(this).select2( {
+        placeholder: $(this).attr('value') || "Search for a user",
+        minimumInputLength: 2,
+        initSelection : function (element, callback) {
+          var data = {id: element.val(), text: element.val()};
+          callback(data);
+        },
+        ajax: { // instead of writing the function to execute the request we use Select2's convenient helper
+          url: "/data/users.json",
```

Comment:
Hard coding this here should work for us for right now.  This is likely a bug upstream.  TODO: Upstream fix.  Either this file should be generated differently or needs some mechanism to handle the relative_url_root.